packages/body/src/sass/module.scss

```diff
@@ -61,7 +61,7 @@ a {
  */
 mark,
 ::selection {
-	background-color: $uikit-colour-Warning-60;
+	background-color: rgba( $uikit-colour-Warning-60, 0.5 );
```

Comment:
Clarifying—this is for when a marked element is selected/highlighted.

Styling can be revisited later.